cryptoshred-spring-boot-autoconfigure/README.md

```diff
@@ -0,0 +1,17 @@
+Module *cryptoshred-spring-boot-autoconfigure*
+==============================================
+
+This maven module provides simple integration of the *cryptoshred* library into Spring Boot.
+
+
+Configuration
+-------------
+
+The following configuration properties exists:
+
+| **Property name**                | **Description** | **Required** | **Default Value** |
+|----------------------------------|-----------------|---------------|----|
+| `cryptoshred.cloud.aws.dynamo.tablename`| name of AWS DynamoDB table to store the subjectId/ key pairs. Only required when module *cryptoshred-cloud-aws* is also used. | yes | - |
+| `cryptoshred.initVector` | initialization vector for encryption/ decryption. Should be a random value. | yes | - |
```

Comment:
This is only required if you want to use the default JDKCryptoEngine.
If you use a different one, you are in charge of initializing it.